Tests/ComposableArchitectureTests/EffectCancellationTests.swift

```diff
@@ -11,6 +11,7 @@ final class EffectCancellationTests: XCTestCase {
   override func tearDown() {
     super.tearDown()
     disposeBag = DisposeBag()
+    cancellationCancellables.removeAll()
```

Comment:
This is in the PF TCA as well. Otherwise when you run all the tests, there are cancellables that hang around between the different test cases. So each test will individually pass, but the run will not (depending on the order).